examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -335,13 +402,32 @@ export class TaskList extends DataObject implements ITaskList {
 			}
 		});
 
+		this._draftData.on("valueChanged", (changed) => {
```

Comment:
Looks like we're registering this same code twice (see lines above)? Merge conflict issue maybe?